frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -903,67 +905,58 @@ export const Search: React.FC<{
 								</Combobox.Item>
 							</Combobox.Group>
 						)}
-						<Combobox.Group
-							className={styles.comboboxGroup}
-							store={comboboxStore}
-						>
-							{!showValues && !showOperators && (
-								<Combobox.GroupLabel store={comboboxStore}>
-									<Box px="10" py="6">
-										<Text color="moderate" size="xxSmall">
-											Recent
-										</Text>
-									</Box>
-								</Combobox.GroupLabel>
-							)}
-							{!showValues &&
+						{!showValues &&
```

Comment:
<img width="693" alt="Screenshot 2024-11-06 at 10 59 42" src="https://github.com/user-attachments/assets/3e594fd9-c525-4036-849a-f8908e83b8b4">

i think this is causing a `0` to render

Reply:
ack